lib/mixins.json

```diff
@@ -89,10 +89,10 @@
     },
     {
       "kind": "org.thingpedia.config.oauth2",
-      "args": [ "client_id", "client_secret" ],
-      "types": [ "String", "String" ],
-      "required": [ false, false ],
-      "is_input": [ true, true ],
+      "args": [ "client_id", "client_secret", "authorize", "get_access_token", "set_state", "get_profile", "profile" ],
+      "types": [ "String", "String", "Entity(tt:url)", "Entity(tt:url)", "Boolean", "Entity(tt:url)", "Array(String)" ],
```

Comment:
what is `profile`? should we have `scope`?

Reply:
profile is an array of keys that you read out of the get_profile call and become part of the device state (which then you can use in the device name/description)